app/models/post.rb

```diff
@@ -14,4 +14,8 @@ def update_posts_counter
   def most_recent_comments
     comments.order(created_at: :desc).limit(5)
   end
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, presence: true, numericality: { only_integer: true }
```

Comment:
- [ ] Here I think you are missing the `greater_than_or_equal_to: 0` as you need to check for it as well, so please fix that issue.

Reply:
@lordkaito Thank you for your feedback